extensions/integration-tests/src/objectExplorer.test.ts

```diff
@@ -11,63 +11,86 @@ import { context } from './testContext';
 import { getBdcServer, TestServerProfile, getAzureServer, getStandaloneServer } from './testConfig';
 import { connectToServer } from './utils';
 import assert = require('assert');
+import { stressify } from '../testfmks/src/stress';
```

Comment:
We tend to avoid naming like `testfmks`, and use `testFrameworks` instead.

Reply:
I am going to call these test Frameworks: Cadre, so I will replace testfmks with cadre everywhere.

Reply:
I am going to rename the test frameworks being created here as 'cadre'.

---
In reply to: [285805493](https://github.com/microsoft/azuredatastudio/pull/5499#discussion_r285805493) [](ancestors = 285805493)

Reply:
Why? We don't encourage codenames, how did you decide on / clear "cadre"? We should be really careful about anything with a codename in code. It's fine to use internally but publicly this can cause trouble (e.g. It's already got a github project at https://github.com/cadre).

I am going to suggest being really boring to avoid problems - `ads-stress` or `azuredatastudio_stress` or `stressframework` if it's a generic thing.

Reply:
Changed to AzDataTest

Reply:
Ok, going with azdatatest, or do you think adstest is preferable to it?

---
In reply to: [286146096](https://github.com/microsoft/azuredatastudio/pull/5499#discussion_r286146096) [](ancestors = 286146096)

Reply:
adstest is what I have finally used.